### PR TITLE
Full support for Bitbucket

### PR DESCRIPTION
**Pull request by nathanaelhoun**
## Changes

- [x] Full support to Bitbucket
- [x] Rename `/circleci config` to `/circleci set-default` for a more idiomatic usage
- [x] Update README.md

image
Two repos with the same name but from different VCS. Links are goods

### CODE REVIEW

**Comment by darkLord19** [comment]
instead of set-default, let's just name it default and also if it's called without any argument then print configured project if exists or help message about how to set config

**Comment by darkLord19** [comment]
Also excellent work :) let's focus on making this and put approve workflow on backburner as it is not needed for submission.

**Comment by nathanaelhoun** [comment]
@darkLord19 Thanks!
You are right. I have renamed it to `/circleci default` as suggested. In the code, I have left `setDefault` to not misunderstanding it is a default command.

File: README.md [hunk 1]
```diff
@@ -104,7 +102,7 @@ Connect to your CircleCI account:
     - /circleci account disconnect          — Disconnect your Mattermost account from CircleCI
 
 Set your default project:
-    - /circleci config <vcs/org-name/project-name> — Set default vcs/org/project for all other commands to use by default
+    - /circleci default [vcs/org-name/project-name] — View your current default project or pass in a new project (<cs/org-name/project-name>
```
Comment (darkLord19) [inline README.md]:
```suggestion
    - /circleci default  — View your currently configured default project
    - /circleci default [vcs/org-name/project-name] — Set new default project by passing value in the form <vcs/org-name/project-name>
```

**Review by darkLord19 — APPROVED** [review]
Great work 👏

**Pull request merged**